src/lib/core/CHIPConfig.h

```diff
@@ -707,6 +707,17 @@
 #define CHIP_CONFIG_MAX_GROUP_CONTROL_PEERS 2
 #endif // CHIP_CONFIG_MAX_GROUP_CONTROL_PEER
 
+/**
+ *  @def CHIP_CONFIG_SLOW_CRYPTO
+ *
+ *  @brief
+ *   When enabled, CASE and PASE setup will proactively send standalone acknowledgements
+ *   prior to engaging in crypto operations.
```

Comment:
Add "because their processing delay far exceeds standalone ACK delay. If CASE and PASE succeeds in less than 300ms, set this to 0".

Suggest making this 0 on Linux/Darwin by default to avoid too many needless standalone ACKs from controllers.

Reply:
Updated config of Linux/Darwin to assume fast crypto on those platforms.